docs/spec/manifest-v2-2.md

```diff
@@ -89,6 +89,11 @@ image manifest based on the Content-Type returned in the HTTP response.
             The os field specifies the operating system, for example
             `linux` or `windows`.
 
+        - **`osversion`** *string*
```

Comment:
Agreed.

If we can come up with a way to namespace the properties as under "os", that would be sufficient. I don't have a suggestion off the top of my head.